examples/nerf/handlers/edit/index.js

```diff
@@ -1,48 +1,48 @@
-'use strict';
-import UserEdit from './container.jsx';
-import {createStore, bindActionCreators} from '@streammedev/flux-store';
-import saveUserReducer, {saveUser} from '../../actions/save-user';
-import deleteReducer, {deleteUser} from '../../actions/delete-user';
-import {changeUserAvatarReducer, changeAvatar} from '../../actions/change-avatar';
-import {changeUserUsernameReducer, changeUsername} from '../../actions/change-username';
-import {changeUserEmailReducer, changeEmail} from '../../actions/change-email';
-import clearNotificationReducer from '../../actions/clear-notification';
+'use strict'
+import UserEdit from './container.jsx'
+import {createStore, bindActionCreators} from '@streammedev/flux-store'
+import saveUserReducer, {saveUser} from '../../actions/save-user'
+import deleteReducer, {deleteUser} from '../../actions/delete-user'
+import {changeUserAvatarReducer, changeAvatar} from '../../actions/change-avatar'
+import {changeUserUsernameReducer, changeUsername} from '../../actions/change-username'
+import {changeUserEmailReducer, changeEmail} from '../../actions/change-email'
+import clearNotificationReducer from '../../actions/clear-notification'
 
 export default function (req, res) {
-	// Create the store with the reducers
-	var store = createStore({
-		deleteUser: deleteReducer,
-		saveUser: saveUserReducer,
-		changeAvatar: changeUserAvatarReducer,
-		changeEmail: changeUserEmailReducer,
-		changeUsername: changeUserUsernameReducer,
-		clearNotification: clearNotificationReducer
-	});
+  // Create the store with the reducers
+  var store = createStore({
+    deleteUser: deleteReducer,
+    saveUser: saveUserReducer,
```

Comment:
It would look like:

```javascript
var store = createStore({
  deleteUser: combineReducers([deleteReducer, otherReducer]),
  saveUser: saveUserReducer
  // ...
})
```